src/hash.cr

```diff
@@ -397,6 +425,8 @@ class Hash(K, V)
   #   key_and_value # => {"foo", "bar"}
   # end
   # ```
+  #
+  # The enumeration follows the order the keys were inserted.
```

Comment:
I'd add it as `NOTE`, ditto all below.

Reply:
I don't think these labels should be overused.